apps/frontend-manage/src/components/evaluation/elements/ChoicesEvaluation.tsx

```diff
@@ -18,29 +25,43 @@ function ChoicesEvaluation({
   chartType,
   showSolution,
 }: ChoicesEvaluationProps) {
+  const [isCollapsed, setIsCollapsed] = useLocalStorage(
```

Comment:
https://www.npmjs.com/package/react-resizable-panels

would be better to use the autosaveId prop as shown here instead of doing it with a manual hook :)